k8s/src/main/scala/io/buoyant/k8s/istio/package.scala

```diff
@@ -1,9 +1,9 @@
 package io.buoyant.k8s
 
 package object istio {
-  val DefaultDiscoveryHost = "istio-manager"
+  val DefaultDiscoveryHost = "istio-pilot"
```

Comment:
Isn't still called the manager as of the 1.6 release?  And should we use the fully qualified name in case linkerd is deployed in a different namespace from istio?